p2p/dialerHTTP.go

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package p2p
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+
+	"github.com/mysteriumnetwork/node/trace"
+
+	"github.com/mysteriumnetwork/node/core/ip"
+	"github.com/mysteriumnetwork/node/core/port"
+	"github.com/mysteriumnetwork/node/firewall"
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/pb"
+
+	"github.com/rs/zerolog/log"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	MsgTypeInit = "init"
+	MsgTypeAck  = "ack"
+)
+
+// NewDialer creates new p2p communication dialer which is used on consumer side.
+func NewDialerHTTP(signer identity.SignerFactory, verifier identity.Verifier, ipResolver ip.Resolver, consumerPinger natConsumerPinger, portPool port.ServicePortSupplier) Dialer {
```

Comment:
Are these files changed or just moved/renamed? Wonder this it's shown as new code